deepchem/models/torch_models/cnn.py

```diff
@@ -0,0 +1,231 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from einops import rearrange
+
+from typing import List, Union, Any, Type
+
+try:
+  from collections.abc import Sequence as SequenceCollection
+except:
+  from collections import Sequence as SequenceCollection
+
+
+class TorchCNN(nn.Module):
+  """A 1, 2, or 3 dimensional convolutional network for either regression or classification.
```

Comment:
This should numpydoc style

https://numpydoc.readthedocs.io/en/latest/format.html